src/main/scala/ing/wbaa/druid/client/DruidHttpClient.scala

```diff
@@ -96,7 +99,10 @@ class DruidHttpClient private (connectionFlow: DruidHttpClient.ConnectionFlowTyp
     Marshal(q)
       .to[RequestEntity]
       .map { entity =>
-        HttpRequest(HttpMethods.POST, uri = druidConfig.url)
+        val requestURL =
+          if (q.queryType == QueryType.SQL) s"${druidConfig.url}sql/" else druidConfig.url
+        logger.info(s"requestURL = ${requestURL}")
```

Comment:
I think this can be a `debug` instead of an `info`. It would also be a good spot to use the `.debug("requestUrl = {}", requestURL)` syntax we inherit from SLF4J, to avoid the string concatenation if the logging is suppressed.